src/utils/repository-url.js

```diff
@@ -24,11 +25,11 @@ module.exports = function(url) {
   url = url.replace('https+git://', 'git+https://');
   url = url.replace('://www.github.com', '://github.com');
 
-  // Resolve detail link
-  var stripDetails = url.match(/https?:\/\/github.com(\/[^\/]+){2,2}/g);
-  if (stripDetails) {
-    url = stripDetails[0];
+  // Ensure there is a protocol (`github-url-to-object` needs it)
+  if (nodeUrl.parse(url).protocol === null) {
+    url = 'https://' + url;
   }
 
-  return githubUrl(url);
+  var githubInfo = githubUrl(url);
+  return githubInfo && githubInfo.https_url || url;
```

Comment:
Wile working on #24, eslint was complaining about this line with `no-mixed-operators`. I think this should be `githubInfo ? githubInfo.https_url : url;`

@josephfrazier can you confirm that please. Thank you